src/components/OrganizationsList/OrganizationsList.tsx

```diff
@@ -1,20 +1,95 @@
 import { OrgObj } from '@/api/cf/cloudfoundry-types';
-import { sortObjectsByParam } from '@/helpers/arrays';
-import { GridList } from '../GridList/GridList';
-import { OrganizationsListItem } from './OrganizationsListItem';
+import { chunkArray, sortObjectsByParam } from '@/helpers/arrays';
+import Link from 'next/link';
+import { MemoryBar } from '@/components/MemoryBar';
+import { formatInt } from '@/helpers/numbers';
+
+export function OrganizationsList({
+  orgs,
+  userCounts,
+  appCounts,
+  memoryAllocated,
+  memoryCurrentUsage,
+  spaceCounts,
+}: {
+  orgs: Array<OrgObj>;
+  userCounts: { [orgGuid: string]: number };
+  appCounts: { [orgGuid: string]: number };
+  memoryAllocated: { [orgGuid: string]: number };
+  memoryCurrentUsage: { [orgGuid: string]: number };
+  spaceCounts: { [orgGuid: string]: number };
+}) {
+  if (!orgs.length) {
+    return <>no orgs found</>;
+  }
 
-export function OrganizationsList({ orgs }: { orgs: Array<OrgObj> }) {
   const orgsSorted = sortObjectsByParam(orgs, 'name');
+  const orgsGrouped = chunkArray(orgsSorted, 3);
 
-  return orgsSorted.length > 0 ? (
-    <GridList>
-      {orgsSorted.map((org) => {
+  return (
+    <div className="margin-y-4">
+      {orgsGrouped.map((orgGoup, groupIndex) => {
         return (
-          <OrganizationsListItem key={`OrgsListItem-${org.guid}`} org={org} />
+          <ul
+            className="grid-row grid-gap-3 usa-card-group"
+            key={`org-group-${groupIndex}`}
+          >
+            {orgGoup.map((org, index) => {
+              return (
+                <li
+                  className="tablet-lg:grid-col-4 margin-bottom-3"
```

Comment:
Just curious: how well would something like `tablet:grid-col-6 tablet-lg:grid-col-4 margin-bottom-3` work here, do you think? I think that’d let us keep the card layout a little longer.

Reply:
It gets a _little_ tight between 880-960px, but perhaps doable. Here's a screenshot at ~900px—with non-breaking spaces added to "at a glance" in the card on the right:

![Screenshot 2024-10-09 at 2 13 03 PM](https://github.com/user-attachments/assets/bba22094-1187-4e52-825b-1539bc05c60d)

Reply:
I think this looks great!

One thought: I wonder if it’s worth vertically stacking “you’re a [role]” and “at a glance” when they start to feel cramped, since their column layout is _much_ less important than the columns for the memory allocation bar.

(No action needed here, sorry — you can resolve this discussion! Just thinking out loud, though I might circle back to this layout tomorrow.)

Reply:
I'll add some right margin to the roles section (as [discussed](https://github.com/cloud-gov/cg-ui/pull/548#discussion_r1794226373) below), and we can keep noodling